flatpak/dev.k8slens.open-lens.yml

```diff
@@ -0,0 +1,54 @@
+app-id: dev.k8slens.open-lens
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
```

Comment:
Why not use 22.08?

Reply:
Good question. For some reason I couldn't get the `node16` extension to install locally for version 22.08, even though it should exist. Not sure if there was something wrong on my end or what the issue there was/is, but, yeah, if that can be made to work, we should use 22.08 instead :+1: